metro.config.js

```diff
@@ -1,22 +1,14 @@
 // Learn more https://docs.expo.io/guides/customizing-metro
 const { getDefaultConfig } = require('@expo/metro-config');
 
-module.exports = (async () => {
-  const {
-    resolver: { assetExts }
-  } = await getDefaultConfig(__dirname);
+const config = getDefaultConfig(__dirname);
 
-  return {
-    transformer: {
-      getTransformOptions: async () => ({
-        transform: {
-          experimentalImportSupport: false,
-          inlineRequires: true
-        }
-      })
-    },
-    resolver: {
-      assetExts: [...assetExts, 'mtl', 'obj', 'vrx']
-    }
-  };
-})();
+config.resolver.sourceExts.push('mtl', 'obj', 'vrx');
```

Comment:
what was `assetExts` is now `sourceExts`?

Reply:
You're right. It was supposed to be `assetExts` 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1136/commits/dcd0f95e12d1e2f4557f848fddb0c382d036eda5